src/custom-types.json

```diff
@@ -236,6 +236,14 @@
   "application/x-7z-compressed": {
     "compressible": false
   },
+  "application/x-arj": {
+    "compressible": false,
+    "extensions": ["arj"],
+    "sources": [
+      "https://en.wikipedia.org/wiki/List_of_archive_formats",
+      "http://www.wussu.com/various/mimetype.htm"
```

Comment:
Thanks everyone for looking into these! I'll give it a few days before I give up on the email to Robert Jung. I think I'll ask in the BBS networks as well (yes, they _still_ exist!).

[Modern Linux file(1)](https://github.com/file/file/blob/daaf6d44768677aca17af780bba0a451fbb69ac8/magic/Magdir/archive) seems to also state `application/x-arj`